roles/software_engineer_2.md

```diff
@@ -2,17 +2,46 @@
 
 Bristol, Manchester & London, UK.
 
-Our Engineers build software that makes our customers happy, they prefer problem solving over task completing, love learning from colleagues, and constantly strive to be nice humans.
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Engineers deliver digital, data and technology outcomes that improve society. They do this by delivering and architecting software.
 
 ## What does the job entail?
 
-We primarily write and deliver custom software for the public sector. We work across central and local government, as well as in health, and our past lies in the technology startup world. Technical excellence for us isn’t about delivering to feature lists. We place a strong emphasis on outcome-based delivery; ensuring our customer’s goals are understood and achieved with the technology we deploy.
+At Made Tech we want to positively impact the future of the country by using technology to improve society. We help public sector organisations deliver quality software to help citizens get more from public services. To do this we work alongside brilliant public servants to modernise technology and accelerate digital delivery.
+
+As an Engineer 2 you will collaboratively lead planning of work and the development of features, coordinating with stakeholders and thier users, and seeking freedback and support when needed, to help influence the approach taken.  You will be comfortable managing complex changes within a single system, and seek support when managing change across multiple systems, ensuring that features meet project outcomes and user needs. 
```

Comment:
I think cut this paragraph - I think 'collaboratively lead' is wording from the senior role, and I think the rest describes behaviours from SFIA 4 + that we wouldn't necessarily expect from an applicant. You could reword but I think it would read fine just cutting it.

Reply:
Cool, reworded to soften the Lead part